### PR TITLE
 [TF] Bring batchNormalized in line with standard batchnorm formula

### PR DESCRIPTION
**Pull request by jekbradbury**
Most batchnorm implementations, including those in Keras and TF, use
vector scale and offsets treated elementwise, not scalars.

### CODE REVIEW

File: stdlib/public/TensorFlow/Ops.swift [hunk 1]
```diff
@@ -1587,8 +1587,8 @@ public extension Tensor where Scalar : BinaryFloatingPoint {
   )
   func batchNormalized(
     alongAxis axis: Int32,
-    offset: Scalar = 0,
-    scale: Scalar = 1,
+    offset: Tensor = Tensor<Scalar>(0),
```
Comment (rxwei) [inline stdlib/public/TensorFlow/Ops.swift]: I believe you can drop the `<Scalar>` signature.

**Review by rxwei — APPROVED** [review]
